frameworks/helloworld/tests/test_resource_refinement.py

```diff
@@ -0,0 +1,25 @@
+import sdk_install as install
+import sdk_utils as utils
+from tests.config import (
+    check_running,
+    DEFAULT_TASK_COUNT,
+    PACKAGE_NAME
+)
+
+def setup_module(module):
+    install.uninstall(PACKAGE_NAME)
+    install.install(
+        PACKAGE_NAME,
+        DEFAULT_TASK_COUNT,
+        service_name=PACKAGE_NAME,
+        additional_options={"service": { "spec_file": "examples/pre-reserved.yml"} })
+
+
+def teardown_module(module):
+    install.uninstall(PACKAGE_NAME)
+
+
+@pytest.mark.sanity
+@pytest.mark.smoke
+def test_install():
+    check_running(PACKAGE_NAME)
```

Comment:
There isn't anything being served there.  It's just one instance of a hello pod.  This isn't really specific to `slave_public` anyway.  It's just a test to see if we can consume statically reserved resources.  `slave_public` happens to be the most convenient and the one needed by EdgeLB.